packages/controllers/src/snaps/SnapController.ts

```diff
@@ -1067,13 +1098,11 @@ export class SnapController extends BaseController<
    */
   public async stopSnap(
     snapId: SnapId,
-    statusEvent:
-      | SnapStatusEvent.stop
-      | SnapStatusEvent.crash = SnapStatusEvent.stop,
+    statusEvent: StatusEvents['type'] & ('STOP' | 'CRASH') = 'STOP',
```

Comment:
It's there mostly for documentation, it states what events are expected in this location, any other will fail.

At the same time it will fail to compile if we change event names without updating it there, which is a nice defensive bonus.